users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -1167,136 +1167,152 @@ def run_specaug_stft_experiments():
-                    "specaug_stft": {"max_feature": 4, "max_feature_num": 2},
+                    "specaug_stft": {
+                        "max_feature": 4,
+                        "max_feature_num": 2,
+                        "frame_size": 400,
+                        "frame_shift": 160,
+                        "fft_size": 512,
+                    },
                 },
-                feature_args=feature_args,
+                feature_args=feature_args_scf,
                 lr_args=lr_args,
                 report_args={"batch_size": "2x5k"},
             ),
-            "bs2x5k_scf_stft_mask_5_8": dict(
+            "bs2x5k_scf_stft_fmask_5_8": dict(
                 returnn_args={
                     **returnn_args,
-                    "specaug_stft": {"max_feature": 8},
+                    "specaug_stft": {"max_feature": 8, "frame_size": 400, "frame_shift": 160, "fft_size": 512},
                 },
-                feature_args=feature_args,
+                feature_args=feature_args_scf,
                 lr_args=lr_args,
                 report_args={"batch_size": "2x5k"},
             ),
-            "bs2x5k_scf_stft_mask_5_15": dict(
+            "bs2x5k_scf_stft_fmask_5_15": dict(
                 returnn_args={
                     **returnn_args,
-                    "specaug_stft": {"max_feature": 15},
+                    "specaug_stft": {"max_feature": 15, "frame_size": 400, "frame_shift": 160, "fft_size": 512},
                 },
-                feature_args=feature_args,
+                feature_args=feature_args_scf,
                 lr_args=lr_args,
                 report_args={"batch_size": "2x5k"},
             ),
-            "bs2x5k_mel_stft_mask_5_8": dict(
+            "bs2x5k_lgm_stft_fmask_5_8": dict(
                 returnn_args={
                     **returnn_args,
-                    "specaug_stft": {"max_feature": 8},
+                    "specaug_stft": {"max_feature": 8, "frame_size": 400, "frame_shift": 160, "fft_size": 512},
                 },
                 feature_args=feature_args_lgm,
                 lr_args=lr_args,
-                report_args={"batch_size": "2x5k",},
+                report_args={
+                    "batch_size": "2x5k",
+                },
             ),
-            "bs10k_mel_stft_mask_5_8": dict(
+            "bs10k_lgm_stft_fmask_5_8": dict(
                 returnn_args={
                     **returnn_args,
-                    "specaug_stft": {"max_feature": 8},
+                    "specaug_stft": {"max_feature": 8, "frame_size": 400, "frame_shift": 160, "fft_size": 512},
                     "batch_size": 10000,
                     "extra_args": {},
                 },
                 feature_args=feature_args_lgm,
                 lr_args=lr_args,
                 report_args={"batch_size": "10k"},
             ),
-            "bs10k_scf_stft_mask_5_8_smaller_frames": dict(
+            "bs10k_scf_stft_fmask_5_8_smaller_frames": dict(
```

Comment:
Was the result good here? If not, we can discard this experiment because I think the hyperparameter selection is rather unintuitive.